.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
 npm-debug.log
+.idea/
```

Comment:
I know, but for my projects I actualy add the `.idea` folder and just exclude `.idea/workspace.xml`